crates/common/consensus/src/constants.rs

```diff
@@ -4,8 +4,8 @@ pub const BASE_REWARDS_PER_EPOCH: u64 = 4;
 pub const BASE_REWARD_FACTOR: u64 = 64;
 pub const BLS_WITHDRAWAL_PREFIX: &[u8] = &[0];
 pub const BYTES_PER_BLOB: usize = BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB;
-pub const BYTES_PER_COMMITMENT: u64 = 48;
 pub const BYTES_PER_FIELD_ELEMENT: usize = 32;
+pub const BYTES_PER_PROOF: usize = 48;
```

Comment:
Can we also use `BYTES_PER_COMMITMENT` and not use `kzg::eip4844`'s constants at all?